client/main.css

```diff
@@ -14,6 +14,9 @@
   --success-green: #99cc33
 }
 
+.remove-underline {
```

Comment:
This class was added to remove the underline from a link. It is not necessary as there's a class named 'link' which should be applied to links to reset their default styles, and it should have been used, just like other links we have.